pkg/platform/images/images.go

```diff
@@ -39,6 +39,8 @@ var StandardDependencies = map[string]bool{
 	"camel:core":   true,
 	"runtime:jvm":  true,
 	"runtime:yaml": true,
+	"mvn:org.apache.camel.k:camel-k-adapter-camel-2:" + defaults.RuntimeVersion: true,
```

Comment:
this seems a little bit of an hack but should work :)

Reply:
Yeah, I plan to remove support for predefined images in 0.3.2..